package.json

```diff
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "chakram": "^1.3.0",
+    "co-mocha": "^1.1.2",
```

Comment:
other libs exist: mocha-generators and mocha-co. I decided to use this one because it was working with simply a `--require` in `mocha.opts` and I found this very easy to setup.

I think we don't get the progressive output anymore, though, but I'm not sure.